kits/evidence-fit/docs/studio-configs/evidence-fit-index.yaml

```
@@ -0,0 +1,60 @@
+      action: index
+      vectorDB: EvidenceFit
+      filters: ''
+      limit: 20
+      primaryKeys:
+        - documentId
+        - strategy
+        - chunkId
+      vectorsField: '{{vectorizeNode_3.output.vectors}}'
+      metadataField: '{{codeNode_2.output.metadata}}'
+      duplicateOperation: overwrite
+      metadataField: '{{codeNode_2.output.metadata}}'
```

YAMLlint (1.37.1)

[error] duplication of key "metadataField" in mapping

(key-duplicates)